.github/workflows/packaging.yml

```diff
@@ -0,0 +1,26 @@
+name: Packaging
+
+on:
+  release:
+    types: [released]
+
+
+jobs:
+  winget:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup winget-create
+        shell: powershell
+        run: |
+          iwr https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
+      - name: Parse release version from github reference
+        run: Set-Item -PATH Env:\GITHUB_CLI_VERSION -Value ${{ github.ref }}.substring(1) 
```

Comment:
Specify the shell in this step

Reply:
Cleaned it up and just compressed it since it didn't need a special step for just setting the version.